contracts/interfaces/IGateway.sol

```diff
@@ -141,6 +149,30 @@ interface IGateway {
 	 */
 	function refund(uint256 _fee, bytes32 _orderId) external returns (bool);
 
+	/**
+	 * @notice Allow Provider to deposit assets.
+	 * @dev Reqirements:
+	 * - The amount must be greater than minimum.
+	 * - The asset must be supported.
+	 * - The provider must approve Gateway contract on `_token` of at least `_amount` before function call
+	 * @param _token The address of the asset.
+	 * @param _amount The amount to be deposited.
+	 * @return bool the deposit is successful.
+	 */
+	function deposit(address _token, uint256 _amount) external returns (bool);
+
+	/**
+	 * @notice Escrowed assets from provider to the sender.
+	 * @param _orderId The ID of the transaction.
+	 * @param _signature The signature of the provider.
+	 * @param _provider The address of the provider.
+	 * @param _senderAddress The address of the sender.
+	 * @param _token The address of the asset.
+	 * @param _amount The amount to be transferred.
+	 * @return bool the withdrawal is successful.
+	 */
+
+
```

Comment:
there's no function here?

Reply:
The function is meant to be for escrow function